ports/lz4/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "lz4",
   "version": "1.9.4",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Lossless compression algorithm, providing compression speed at 400 MB/s per core.",
   "homepage": "https://github.com/lz4/lz4",
-  "license": "BSD-2-Clause AND GPL-2.0-only",
+  "license": "GPL-2.0-or-later",
```

```suggestion
  "license": "BSD-2-Clause",
```
is what I see. vcpkg only installs the lib.
GPL is in programs, tests, examples.
https://tracker.debian.org/media/packages/l/lz4/copyright-1.9.4-2